src/native/rollup.config.plugins.js

```diff
@@ -0,0 +1,152 @@
+//! Licensed to the .NET Foundation under one or more agreements.
+//! The .NET Foundation licenses this file to you under the MIT license.
+
+import { createHash } from "crypto";
+import { readFile, writeFile, mkdir } from "fs/promises";
+import virtual from "@rollup/plugin-virtual";
+import * as fs from "fs";
+import * as path from "path";
+import terser from "@rollup/plugin-terser";
+
+import { isContinuousIntegrationBuild, gitHash } from "./rollup.config.defines.js"
+
+export const terserPlugin = (terserOptions) => {
+    let { compress, mangle } = terserOptions || {};
+    compress = compress || {};
+    mangle = mangle || {};
+    return terser({
+        ecma: "2020",
```

Comment:
I see `ES2020` in the `tsconfig.json` below. Do they need to stay in sync? If so, can we have some assert or other marker to let us know when they inevitably drift? I'm concnered about someone spending hours trying to figure out why something is broken in an odd way.

Reply:
This is target language level. Ideally they should stay in sync. 
In practice, this is not a problem very often.
I will add comment about the other place in echo of them.